modular_skyrat/modules/oversized/code/door.dm

```diff
@@ -1,11 +1,11 @@
-/obj/machinery/door/Initialize(mapload)
+/obj/machinery/door/airlock/Initialize(mapload)
 	. = ..()
 	var/static/list/loc_connections = list(
 		COMSIG_ATOM_ENTERED = .proc/on_entered,
 	)
 	AddElement(/datum/element/connect_loc, loc_connections)
 
-/obj/machinery/door/proc/on_entered(datum/source, atom/movable/crossed_atom)
+/obj/machinery/door/airlock/proc/on_entered(datum/source, atom/movable/crossed_atom)
 	SIGNAL_HANDLER
 	if(HAS_TRAIT(crossed_atom, TRAIT_OVERSIZED) && ishuman(crossed_atom))
 		var/mob/living/carbon/human/crossing_human = crossed_atom
```

Comment:
That means that they wouldn't hit their head on regular doors too, which I think is something we'd like to keep.

Reply:
Regular doors don't really come up that much to be fair, and I'd have to change their path or some nonsense to have them be affected but not windoors/firelocks.

Reply:
You could also just have the element either removed, or simply neuter the on_entered proc.